### ISSUE TITLE
Many Draco models (.drc) kill browser

### ISSUE THREAD

**Issue by EstebanFuentealba** [mrdoob/three.js#22944]
**Describe the bug**

I Drop 341 Draco Models to three editor browser and kill it (tested with autosave and without autosave)

[![Bug](https://i.giphy.com/media/nMTMdgekDBzCkVOndy/giphy.webp)](https://www.youtube.com/watch?v=tWCvGqKwV9w)

**To Reproduce**

1. Go to 'https://threejs.org/editor/'
2. Drop this models https://www.mediafire.com/file/51t4e10weaxsd81/boxes.zip/file
3. See error

**Expected behavior**

render all models or warn of the maximum of resources used

**Platform:**

 - Device: Desktop
 - OS: MacOS M1
 - Browser: Chrome
 - Three.js version: r135

**Comment by Mugen87** [comment]
> I Drop 341 Draco Models to three editor

Do you mind explaining your use case? I mean are you doing this just for fun or do you have a realistic use case?

In will be hard to detect from JavaScript resource limitations of the underlying hardware. Working with heuristics (like the number of models) is in general problematic because browsers are used on such a wide variety of devices.

**Comment by mrdoob** [comment]
I just tried dropping 5767543 draco files and they load fine 😁

**Comment by mrdoob** [comment]
Anyway... Yes, browser tabs crash when they run out of memory.

**Comment by EstebanFuentealba** [comment]
> > I Drop 341 Draco Models to three editor
> 
> Do you mind explaining your use case? I mean are you doing this just for fun or do you have a realistic use case?
> 
> In will be hard to detect from JavaScript resource limitations of the underlying hardware. Working with heuristics (like the number of models) is in general problematic because browsers are used on such a wide variety of devices.
> 

yes, I extended this editor to show industrial equipment that has various 3d models and add some extra functions... I used those drc because it was the best performing format

**Comment by Mugen87** [comment]
@EstebanFuentealba Does this issue still happen on `dev`? You can test with:

https://raw.githack.com/mrdoob/three.js/dev/editor/index.html

**Comment by EstebanFuentealba** [comment]
> @EstebanFuentealba Does this issue still happen on `dev`? You can test with:
> 
> https://raw.githack.com/mrdoob/three.js/dev/editor/index.html

Good 👏, thanks, now the browser does not crash
